webclient/app/src/app/commons/table/OverviewTable.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
```

Comment:
Wird diese Komponente irgendwo verwendet?

Reply:
Jepp - im Template. Wenn die commons ausgelagert werden, muss diese Komponente drin sein.

Reply:
Ahh,würde es dann nicht mehr Sinn machen, da schonmal das seperat auszulagern? Also in einem seperaten Repo? Weil in dem Projekt brauchen wir das ja sonst nicht...